src/app/tests/suites/certification/Test_TC_BRBINFO_2_1.yaml

```diff
@@ -140,21 +140,16 @@ tests:
       response:
           value: ProductNameValue
 
-    - label:
-          "Step 14: TH reads attribute ID 4 from the DUT (matches in ID to
-          ProductID in the parent cluster, but is absent on the
-          BridgedDeviceBasicInformation cluster)."
+    - label: "Step 14: TH reads ProducID from the DUT"
       PICS: BRBINFO.S
```

Comment:
Does this PICS need to be:

`BRBINFO.S.A0004`?

Reply:
ack